includes/gateways/stripe/includes/filters.php

```diff
@@ -48,18 +48,20 @@ function give_stripe_get_payment_txn_id_fallback( $payment_id ) {
 /**
  * This function is used to add Stripe credentials to GiveWP form.
  *
- * @param array  $form_html_tags Form HTML tags.
+ * @unreleased Return $form_html_tags instead of false when Stripe gateway isn't enabled
+ * @since 2.7.0
+ *
  * @param object $form           Form Object.
  *
- * @since 2.7.0
+ * @param array  $form_html_tags Form HTML tags.
  *
  * @return array|bool
  */
 function give_stripe_form_add_data_tag_keys( $form_html_tags, $form ) {
 
 	// Must have a Stripe payment gateway active.
-	if ( ! give_stripe_is_any_payment_method_active() ) {
-		return false;
+    if ( ! give_stripe_is_any_payment_method_active()) {
```

Comment:
I'm debating if we should update the `give_stripe_is_any_payment_method_active` method as well.  It seems to be used in a few other locations, but is synonymously used with the v2 Stripe gateways.

If we left it alone, it will only refer to the option based gateway tab - which is where all the v2 stripe gateways live.  That could avoid loading things that are not necessary for v3 Stripe.

If we updated it, the stripe payment element would be included. Although, that might cause some side effects - like in this situation it would add stripe data to a v2 form even though the Stripe payment element is only used on v3 forms.

Since Stripe Payment Element does use some legacy code like the webhooks, it's possible this function could hinder some overlooked functionality.

🤔

Reply:
@jonwaldstein If I understand all that you said, [this commit](344faced1fcaea18a3444c9ef4bf55e747fac864) should prevent unexpected returns for this function. Let me know your thoughts on it.

Reply:
@glaubersilva haha nice, as a note - I prefer not to access superglobals like that in functions because I feel that sort of thing should be elevated outside of the function - closer to the actual request.  

This did spark an idea though, how about we just add a filter to the return value `give_stripe_is_any_payment_method_active` including the `$gateways`, and `$stripePaymentMethods`.

Then if/when we ever need to include the Stripe Payment Element we can just use the filter.  (That's when I would allow reaching for superglobals)

Reply:
Resolved in 8c7470136e2b43da0c43b6a64d02063731bfec5f